src/mumble/Log.cpp

```diff
@@ -333,35 +333,32 @@ void Log::clearIgnore() {
 	qmIgnore.clear();
 }
 
+unsigned int Log::estimatedDataURLImageSize(const QByteArray &buf) {
```

Comment:
I'd prefer if this was just a static function in this file. Then we wouldn't pollute the Log class with it. (Doc should just be in here as well.)